.github/workflows/draft_without_issue.yml

```diff
@@ -0,0 +1,32 @@
+name: Convert PRs to Drafts without Linked Issue
+
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  checkLinkedIssue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        id: set-result
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prDescription = context.payload.pull_request.body;
+            const issueMatcher = /[cC]loses:\s*#(\d+)/;
```

Comment:
"Fixes" is also valid -- we shouldn't rely on regex unless we really have no other choice...